src/runtime/propensity-test.js

```diff
@@ -182,6 +201,14 @@ describes.realWin('Propensity', (env) => {
       })
     ).to.throw('Event data must be an Object(all_offers)');
 
+    //ensure it rejects invalid `active` values
```

Comment:
nits: Add space after //, use sentence case, end with period.

Reply:
Ah yes, that's more readable